index.html

```diff
@@ -26,22 +28,151 @@
+                                    <p class="cardtext">Profesional Art Printing DataA daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                                    <button type="button" class="btn3 btn-primary btn-lg">HTML</button>
+                                        <button type="button" class="btn3 btn-primary btn-lg">BootStrap</button>
+                                        <button type="button" class="btn3 btn-primary btn-lg">Ruby</button>
+                                        <div class ="footer"><button class = "projects btn-primary btn-lg">See project</button></div>
+                                    </div></li>
+                                        <li class="cardlist"> <div id="cwaction1" class="cwaction1">
+                                            <h2 class="card1">Professional Art Printing Data</h2>
+                                            <p class="cardtext">Profesional Art Printing DataA daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                                            <button type="button" class="btn3 btn-primary btn-lg">HTML</button>
+                                                <button type="button" class="btn3 btn-primary btn-lg">BootStrap</button>
+                                                <button type="button" class="btn3 btn-primary btn-lg">Ruby</button>
+                                                <div class ="footer"><button class = "projects btn-primary btn-lg">See project</button></div>
+                                            </div></li>
+                                                <li class="cardlist"> <div id="cwaction1" class="cwaction1">
+                                                    <h2 class="card1">Professional Art Printing Data</h2>
+                                                    <p class="cardtext">Profesional Art Printing DataA daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                                                    <button type="button" class="btn3 btn-primary btn-lg">HTML</button>
+                                                        <button type="button" class="btn3 btn-primary btn-lg">BootStrap</button>
+                                                        <button type="button" class="btn3 btn-primary btn-lg">Ruby</button>
+                                                        <div class ="footer"><button class = "projects btn-primary btn-lg">See project</button></div>
+                                                    </div></li>
+        </ul>
+    </section>
+    <section id="about-section" class="about-section">
+        <div class="about-introduction-content">
+          <h1 class="about-heading">About me</h1>
```

Comment:
- [ ] Kindly consider to change this from h1 to `h2 or h3` because we should only have one h1 in each webpage to follow best semantics and practices.